ViAn/ViAn.pro

```diff
@@ -15,7 +15,9 @@ TEMPLATE = app
 #
 # GENERAL
 #
-SOURCES += main.cpp
+SOURCES += main.cpp \
+    Filehandler/videoproject.cpp \
+    Filehandler/bookmark.cpp
 
```

Comment:
You've forgotten to put the sources and headers in the correct places n the pro-file.